docs/en/administration/sync.md

```diff
@@ -1,4 +1,243 @@
 ---
-sidebar_label: Sync
+sidebar_label: Synchronization
 ---
-# Sync
+
+# Migrate and Synchronize Data across Clouds with JuiceFS Sync
+
+The `sync` subcommand of JuiceFS is a full-featured data synchronization utility that can synchronize or migrate data concurrently with multiple threads between all [object storages JuiceFS supports](../reference/how_to_setup_object_storage.md). It can be used to migrate data not only between _object storage_ and _JuiceFS_, but also between _object storages_ in different clouds or regions. In addition, it can also be used to synchronize local directories, access remote directories through SSH, HDFS, WebDAV, etc., quite similar to `rsync`. It also provides advanced features such as full synchronization, incremental synchronization, and conditional pattern matching.
```

Comment:
```suggestion
The `sync` subcommand of JuiceFS is a full-featured data synchronization utility that can synchronize or migrate data concurrently with multiple threads between all [object storages JuiceFS supports](../reference/how_to_setup_object_storage.md). It can be used to migrate data not only between _object storage_ and _JuiceFS_, but also between _object storages_ in different clouds or regions. In addition, similar to `rsync`, the JuiceFS `sync` subcommand can also be used to synchronize local directories and access remote directories through SSH, HDFS, WebDAV, etc.. It also provides advanced features such as full synchronization, incremental synchronization, and conditional pattern matching.
```
原本已经翻译的很好啦，其实不改动也是可以的 :) 
不过个人感觉句式有点点单一，所以我稍微改了一下某个句子的结构，并且把段中的一个"it" 换成了 the `sync` subcommand，重新强调一下我们的主语 :P  你看看有没有必要哈